core/commands/storage.go

```diff
@@ -326,6 +282,72 @@ var storageUploadResponseChallengeCmd = &cmds.Command{
 	},
 }
 
+func p2pCall(n *core.IpfsNode, pid peer.ID, api string, arg ...string) ([]byte, error) {
+	remoteCall := &remote.P2PRemoteCall{
+		Node: n,
+		ID:   pid,
+	}
+	var args []string
+	for _, str := range arg {
+		args = append(args, str)
+	}
+	return remoteCall.CallGet(api, args)
```

Comment:
Why don't you make `CallGet` take `arg ...string` instead? It makes more sense not to do any arg parsing outside `CallGet`

Reply:
You are right, totally redundant here.